app/src/main/java/de/xikolo/config/Feature.kt

```diff
@@ -16,10 +16,11 @@ object Feature {
 
     @JvmField
     val PIP =
-        Build.VERSION.SDK_INT >= 26 && App.instance.packageManager
+        SDK_INT >= 26 && App.instance.packageManager
             .hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE)
 
-    val SHORTCUTS = Build.VERSION.SDK_INT >= 25
+    val SHORTCUTS = SDK_INT >= 25
+    val FOREGROUNDS = SDK_INT >= 23
```

Comment:
Maybe name it a bit more specific like `VIEW_FOREGROUNDS`.

Reply:
I would build the layout with a `FrameLayout`, so `android:foreground` works API21+ and these "Feature" checks are not necessary.
See https://stackoverflow.com/questions/52329249/why-doesnt-androidforeground-attribute-work